test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -392,18 +419,25 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
             later_tenant_usage < du_by_timeline[tenant]
         ), "all tenants should have lost some layers"
 
+    warmed_up = later_du_by_timeline[our_tenant]
     assert (
-        later_du_by_timeline[our_tenant] > 0.5 * du_by_timeline[our_tenant]
-    ), "our warmed up tenant should be at about half capacity, part 1"
+        warmed_up > 0.5 * du_by_timeline[our_tenant]
+    ), f"our warmed up tenant should be at about half capacity but was {human_bytes(warmed_up)} expected more than {human_bytes(0.5 * du_by_timeline[our_tenant])}"
     assert (
         # We don't know exactly whether the cold tenant needs 2 or just 1 env.layer_size wiggle room.
         # So, check for up to 3 here.
-        later_du_by_timeline[our_tenant]
+        warmed_up
         < 0.5 * du_by_timeline[our_tenant] + 3 * env.layer_size
     ), "our warmed up tenant should be at about half capacity, part 2"
+
+    other_size = later_du_by_timeline[other_tenant]
+
+    log.info(
+        f"later_du_by_timeline[other_tenant] = {other_size} ({human_bytes(other_size)}), before_relax = {other_size < 2 * env.layer_size}"
+    )
```

Comment:
```suggestion
    log.info(
        f"later_du_by_timeline[other_tenant] = {other_size} ({human_bytes(other_size)})"
    )
```

Or is this really necessary... Maybe should:
1. read these both
2. calculate limits
3. log them all
4. assert

Reply:
Done in later.